latest/index.bs

```diff
@@ -177,308 +177,99 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
 
 Tables {#table-layout}
 ----------------------
-The following describes the expected layout for tabular data.
-OME-NGFF tables are compatible with the [AnnData model](https://github.com/scverse/anndata).
+A table is a 2-dimensional data structure consisting of rows and columns. 
+Tables are an intuitive way of storing arbitrary data or metadata that are 
+highly structured, potentially large, and are in some way associated with an image.
+
+In OME-NGFF, a table is a Zarr group containing zero, one, or more Zarr arrays, where each
+array represents one column of the table. Columns are ordered, and each column in a 
+table MUST have the same number of rows. While the table itself MUST be 2-dimensional, 
+the columns need not be. 1-dimensional columns will be typical. Nevertheless, there 
+may be use cases in which it is sensible to conceptualize a 2D, 3D, or higher-dimensional 
+array as a single column. A row of the table is the set of arrays obtained by slicing all
+columns at the same index in the 0-th dimension.
+
+Tables in an OME-NGFF file are located in the `tables/` directory, in the root of an image's
+Zarr group, alongside the labels/ directory if one is present. The .zattrs file in each
+table of `tables/` MUST contain the "annotated-data" property, which is a JSON array of
+Zarr array paths and dimensions (0-based indexing), as shown below:
+
+```json
```

Comment:
I didn't understood this example. Could you please elaborate more on this? Why do you need to tell which dimension of an image is being annotated for each table?

Furthermore, does this allow to annotate labels? For a table annotating labels I would just put in the metadata that the table refers to the Zarr `labels` group, which do you need to specify which one of the yx dimension is being annotated?

Reply:
By requiring the image and dimension that is annotated by a table, I see two advantages compared to having a fixed naming schema:
1. We are free to name the tables whatever we like, e.g., `obs` for the 0-th dimension of an array `X` and `var` for the first dimension of the same array (this may also answer @will-moore's questions of how to store the `var` array), without imposing this naming scheme on others, who might want to choose a more descriptive name for a more specific use case.
2. We can annotate the same image/dimension pair by more than one table. Vice versa, one table can annotate multiple image/dimension pairs, in which case the 'orientation' of the table comes from the annotated dimension, i.e., the table can annotated the columns of one image and the rows of another one.

Reply:
Thanks for the comment. I now understood the purpose, but I want to point out some ambiguity in the terminology that you used; I think that this causes some confusion in the proposal and that addressing this can make the proposal clearer.

From your message above it seems like that the `obs` is the table and `X` is the image. In the `AnnData` proposal `X` is never an image. `X` is always a matrix of values, i.e. the table is "`X` + `obs` + `var`". `obs` alone is not considered a table. 

In the `AnnData` proposal, if we have a segmentation mask (labels tensor) called `lab`, the way to annotated it would be by a matrix `X` containing the annotations and by a `obs` dataframe containing the two columns that tell how to map the indices of the segmentation mask and the indices of the annotation matrix `X`. So the table "`X` + `obs`" annotates the labels `lab`.

This is why I was confused by the new proposal, because what you call the table (`obs`) should now tell which dimension of `X` is being annotated, but `X` is not the image, it's the annotation table itself.

Reply:
With this comment I don't want to say that `obs` + `var` + `X` = `table` is better or worse. It has it's advantages and it's disadvantages. I just want to point out the ambiguity in the proposal since it confused me and could confuse other readers.

Reply:
I think the second sentence of the Tables spec ("Tables are an intuitive way of storing...") contributes to this confusion. I would replace it with this: "Tables are an intuitive way of organizing data or metadata consisting of variables and records, which are often organized as columns and rows, respectively."

I agree with Michael that I am pretty happy with our definition of table as it is in the minimal spec, though I am also happy to make changes that will minimize confusion. In relational databases, if you have two tables that together tell a story, we treat them as two tables, and the schema describes how they relate to one another. I think if AnnData wants to conceptualize multiple 2D data structures as a single table, that's fine, but we want to create a spec that is generic and intuitive for people who've never encountered AnnData.